super_calendar/views/super_calendar_view.xml

```diff
@@ -81,16 +90,20 @@
             <field name="name">super_calendar_form</field>
             <field name="model">super.calendar</field>
             <field name="arch" type="xml">
-                <form string="Configurator">
-                    <field name="name" readonly="1"/>
-                    <field name="date_start" readonly="1"/>
-                    <!--<field name="date_stop" readonly="1"/>-->
-                    <field name="duration" readonly="1"/>
-                    <field name="user_id" readonly="1"/>
-                    <field name="configurator_id" readonly="1"/>
-                    <field name="model_id" readonly="1"/>
-                    <field name="model_description" readonly="1"/>
-                    <field name="res_id"/>
+                <form string="Calendar">
+                    <sheet>
+                        <group>
+                            <field name="name" readonly="1"/>
+                            <field name="date_start" readonly="1"/>
+                            <!--<field name="date_stop" readonly="1"/>-->
```

Comment:
Now would be a good time to get rid of this commented out field.
If there is a good reason to keep it, then there should be a comment here to tell us why it's done this way.